conf.d/yarn.yaml.example

```diff
@@ -17,7 +17,12 @@ instances:
     # A Required friendly name for the cluster.
     # cluster_name: MyYarnCluster
 
+    # Tags from running applications that will be applied for application metrics, set as app_key:datadog_tag
+    #
+     application_tags:
+       - name:app_name
+
     # Optional tags to be applied to every emitted metric.
-    # tags:
+    # extra_tags:
```

Comment:
I understand why you changed it to `extra_tags`, but for consistency reasons (all the checks use `tags`) it would be better to keep `tags`.